tasks/apply-mapping/README.md

```diff
@@ -29,6 +29,11 @@ You can also expand image labels, e.g. "{{ labels.mylabel }}" -> The value of im
 | dataPath          | Path to the JSON string of the merged data to use in the data workspace                      | No       | -             |
 | failOnEmptyResult | Fail the task if the resulting snapshot contains zero components                             | Yes      | false         |
 
+## Changes in 1.8.0
+* Enhanced tag generation by adding support for the {{ incrementer }} variable:
+  * The {{ incrementer }} variable automatically finds the highest existing incremented tag
```

Comment:
This also needs to be listed after line 20

Reply:
Done, thanks!